R/bbr.R

```diff
@@ -338,6 +351,18 @@ bbi_help <- function(.cmd_args=NULL) {
 #'
 #' @importFrom yaml read_yaml write_yaml
 #' @importFrom fs dir_exists
+#'
+#' @examples
+#' \dontrun{
+#'
+#' # NONMEM executable found at `/opt/NONMEM/nm75/run`:
+#'
+#' bbi_init(
+#'    .dir = "inst/model/nonmem/basic",
```

Comment:
From the bbr checkout, using this `inst/` path is of course handy to test, but that's an unusual directory from the user perspective.  What about just using a `MODEL_DIR` stand-in?  That would align with what's used expo1.

Reply:
Yeah I think that makes sense. Addressed.